gchart-parent/gchart-examples/src/main/webapp/META-INF/context.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Context path="/wicket-gcharttrial"/>
```

Comment:
My IDE (netbeans 8.2) does not deploy to Tomcat without  context.xml, don't know why. Just leave it for now.